templates/templates.go

```diff
@@ -20,6 +20,8 @@ type TemplateType string
 const (
 	// Snippet will mark a template as a part of a file.
 	Snippet TemplateType = "snippet"
+	// Line is a template for a single line in a file.
+	Line TemplateType = "line"
```

Comment:
It's not exactly a single line as we will be prepending lines on it. Can you clarify the comment?